src/components/common/infocolourBar/InfoColourbar.tsx

```diff
@@ -2,20 +2,21 @@ import React from 'react'
 import './infoColourBar.scss'
 
 interface InfoColourBarType {
-    message: string
+    message: React.ReactNode
     classname: string
     Icon
     iconClass?: string
+    iconSize?: string // E.g. icon-dim-16, icon-dim-20, etc.
```

Comment:
can we take number in this instead of whole classname. as iconSize doesn't resemble to icon-size class

Reply:
Done